adserver/views.py

```diff
@@ -823,6 +825,55 @@ def get_success_url(self):
         )
 
 
+class PublisherPayoutListView(PublisherAccessMixin, UserPassesTestMixin, ListView):
+
+    """List of publisher payouts."""
+
+    model = PublisherPayout
+    template_name = "adserver/publisher/payouts.html"
+
+    def get_context_data(self, **kwargs):  # pylint: disable=arguments-differ
+        context = super().get_context_data(**kwargs)
+
+        payouts = self.get_queryset()
+        total = payouts.aggregate(
+            total=models.Sum("amount", output_field=models.FloatField())
```

Comment:
Seems odd to use a Float to count money, is there not Decimal support here?

Reply:
The underlying field where the payout is stored is an exact decimal field. Only when it is aggregated is it rounded into a float field. I didn't actually test whether a decimal field was supported here.

Reply:
Yea -- I think we need to get the website built out. Having a place for blog posts, policy, etc. seems more and more important.

Reply:
A decimal field will work fine for aggregating.